### PR TITLE
Add missing domain

### PR DESCRIPTION
**Pull request by aleeks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add missing domain
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | -
| How to test?  | -

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
* Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: themes/classic/templates/customer/order-slip.tpl [hunk 1]
```diff
@@ -29,7 +29,7 @@
 {/block}
 
 {block name='page_content'}
-  <h6>{l s='Credit slips you have received after canceled orders' d='Shop.Theme.Customeraccount'}.</h6>
+  <h6>{l s='Credit slips you have received after canceled orders' d='Shop.Theme.Customeraccount.'}</h6>
```
Comment (antoin-m) [inline themes/classic/templates/customer/order-slip.tpl]: Why? :walking_man:

File: controllers/admin/AdminShippingController.php [hunk 2]
```diff
@@ -43,72 +43,72 @@ public function __construct()
...
                         'suffix' => Configuration::get('PS_WEIGHT_UNIT'),
                         'cast' => 'floatval',
                         'type' => 'text',
                         'validation' => 'isUnsignedFloat'),
                 ),
                 'description' =>
                     '<ul>
-						<li>'.$this->l('If you set these parameters to 0, they will be disabled.').'</li>
-						<li>'.$this->l('Coupons are not taken into account when calculating free shipping.').'</li>
+						<li>'.$this->trans('If you set these parameters to 0, they will be disabled.', array(), 'Admin.Shipping.Feature').'</li>
```
Comment (AlexEven) [inline controllers/admin/AdminShippingController.php]: Admin.Shipping.Help

File: controllers/admin/AdminShippingController.php [hunk 3]
```diff
@@ -43,72 +43,72 @@ public function __construct()
...
+						<li>'.$this->trans('Coupons are not taken into account when calculating free shipping.', array(), 'Admin.Shipping.Feature').'</li>
```
Comment (AlexEven) [inline controllers/admin/AdminShippingController.php]: Admin.Shipping.Help

File: controllers/admin/AdminShippingController.php [hunk 4]
```diff
@@ -43,72 +43,72 @@ public function __construct()
...
+                        'title' => $this->trans('Default carrier', array(), 'Admin.Shipping.Feature'),
+                        'desc' => $this->trans('Your shop\'s default carrier', array(), 'Admin.Shipping.Help'),
                         'cast' => 'intval',
                         'type' => 'select',
                         'identifier' => 'id_carrier',
                         'list' => array_merge(
                             array(
-                                -1 => array('id_carrier' => -1, 'name' => $this->l('Best price')),
-                                -2 => array('id_carrier' => -2, 'name' => $this->l('Best grade'))
+                                -1 => array('id_carrier' => -1, 'name' => $this->trans('Best price', array(), 'Admin.Global')),
```
Comment (AlexEven) [inline controllers/admin/AdminShippingController.php]: Both "Best price" and "Best grade" should stay Admin.Shipping.Feature as it's related to shipping and not appearing anywhere else.

File: controllers/admin/AdminShippingController.php [hunk 5]
```diff
@@ -43,72 +43,72 @@ public function __construct()
...
+                                -2 => array('id_carrier' => -2, 'name' => $this->trans('Best grade', array(), 'Admin.Global')),
                             ),
                             Carrier::getCarriers((int)Configuration::get('PS_LANG_DEFAULT'), true, false, false, null, Carrier::ALL_CARRIERS))
                     ),
                     'PS_CARRIER_DEFAULT_SORT' => array(
-                        'title' => $this->l('Sort by'),
-                        'desc' => $this->l('This will only be visible in the front office.'),
+                        'title' => $this->trans('Sort by', array(), 'Admin.Actions'),
+                        'desc' => $this->trans('This will only be visible in the front office.', array(), 'Admin.Shipping.Feature'),
```
Comment (AlexEven) [inline controllers/admin/AdminShippingController.php]: Admin.Shipping.Help

File: controllers/admin/AdminShippingController.php [hunk 6]
```diff
@@ -43,72 +43,72 @@ public function __construct()
...
                         'cast' => 'intval',
                         'type' => 'select',
                         'identifier' => 'value',
                         'list' => $carrier_default_sort
                     ),
                     'PS_CARRIER_DEFAULT_ORDER' => array(
-                        'title' => $this->l('Order by'),
-                        'desc' => $this->l('This will only be visible in the front office.'),
+                        'title' => $this->trans('Order by', array(), 'Admin.Actions'),
+                        'desc' => $this->trans('This will only be visible in the front office.', array(), 'Admin.Shipping.Feature'),
```
Comment (AlexEven) [inline controllers/admin/AdminShippingController.php]: Admin.Shipping.Help

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/tab-module-line.html.twig [hunk 7]
```diff
@@ -42,14 +42,14 @@
...
         {% elseif module.attributes.type is not empty and module.attributes.type == 'addonsPartner' %}
-        - <span class="module-badge-partner help-tooltip text-warning" data-title="{{ 'This module is available for free thanks to our partner.'|trans({}) }}"><i class="icon-pushpin"></i> <small>{{ 'Official'|trans({}) }}</small></span>
+        - <span class="module-badge-partner help-tooltip text-warning" data-title="{{ 'This module is available for free thanks to our partner.'|trans({}, 'Admin.Modules.Feature') }}"><i class="icon-pushpin"></i> <small>{{ 'Official'|trans({}, 'Admin.Modules.Feature') }}</small></span>
         {% elseif module.attributes.id is defined and module.attributes.id >= 0 %}
           {% if module.attributes.version_addons is defined and module.attributes.version_addons %}
-            <span class="label label-warning">{{ 'Need update'|trans({}) }}</span>
+            <span class="label label-warning">{{ 'Need update'|trans({}, 'Admin.Modules.Feature') }}</span>
```
Comment (AlexEven) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/tab-module-line.html.twig]: Admin.Modules.Notification

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/tab-module-line.html.twig [hunk 8]
```diff
@@ -42,14 +42,14 @@
         {{ module.attributes.displayName }}
         <small class="text-muted">v{{ module.attributes.version }} - by {{ module.attributes.author }}</small>
         {% if module.attributes.type is not empty and module.attributes.type == 'addonsBought' %}
-        - <span class="module-badge-bought help-tooltip text-warning" data-title="{{ 'You bought this module on PrestaShop Addons. Thank You.'|trans({}) }}"><i class="icon-pushpin"></i> <small>{{ 'Bought'|trans({}) }}</small></span>
+        - <span class="module-badge-bought help-tooltip text-warning" data-title="{{ 'You bought this module on PrestaShop Addons. Thank You.'|trans({}, 'Admin.Modules.Feature') }}"><i class="icon-pushpin"></i> <small>{{ 'Bought'|trans({}, 'Admin.Modules.Feature') }}</small></span>
```
Comment (AlexEven) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/tab-module-line.html.twig]: Admin.Modules.Help

(but we should comment all this, I think it's no longer used)

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/tab-module-line.html.twig [hunk 9]
```diff
@@ -42,14 +42,14 @@
...
         {% elseif module.attributes.type is not empty and module.attributes.type == 'addonsMustHave' %}
-        - <span class="module-badge-popular help-tooltip text-primary" data-title="{{ 'This module is available on PrestaShop Addons.'|trans({}) }}"><i class="icon-group"></i> <small>{{ 'Popular'|trans({}) }}</small></span>
+        - <span class="module-badge-popular help-tooltip text-primary" data-title="{{ 'This module is available on PrestaShop Addons.'|trans({}, 'Admin.Modules.Feature') }}"><i class="icon-group"></i> <small>{{ 'Popular'|trans({}, 'Admin.Modules.Feature') }}</small></span>
```
Comment (AlexEven) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/tab-module-line.html.twig]: Admin.Modules.Help

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/tab-module-line.html.twig [hunk 10]
```diff
@@ -42,14 +42,14 @@
...
         {% elseif module.attributes.type is not empty and module.attributes.type == 'addonsPartner' %}
-        - <span class="module-badge-partner help-tooltip text-warning" data-title="{{ 'This module is available for free thanks to our partner.'|trans({}) }}"><i class="icon-pushpin"></i> <small>{{ 'Official'|trans({}) }}</small></span>
+        - <span class="module-badge-partner help-tooltip text-warning" data-title="{{ 'This module is available for free thanks to our partner.'|trans({}, 'Admin.Modules.Feature') }}"><i class="icon-pushpin"></i> <small>{{ 'Official'|trans({}, 'Admin.Modules.Feature') }}</small></span>
```
Comment (AlexEven) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/tab-module-line.html.twig]: Admin.Modules.Help

**Review by AlexEven — COMMENTED** [review]
Thank you very very much @aleeks!!

**Comment by aleeks** [comment]
Done on 1.7.1.x

**Pull request closed without merging**